systems/controllers/osc/osc_user_defined_traj.h

```diff
@@ -0,0 +1,41 @@
+// TODO(yminchen):
```

Comment:
If this isn't going to be included in the PR, best to remove it for now.

Reply:
Yes, I'll move the reduced order model related code to my branch